mods/ts/rules/nod-vehicles.yaml

```diff
@@ -175,6 +175,8 @@ TTNK:
 		RequiresCondition: undeployed
 	RevealOnFire:
 		ArmamentNames: primary, deployed
+	RenderVoxels:
+		Scale: 11.5
```

Comment:
12 is slightly too big for these two at least.  How about I change the default scale to 12, then keep the 11.5 here?